### ISSUE TITLE
View clients page

### ISSUE THREAD

**Issue by joeldenning** [JustUtahCoders/comunidades-unidas-internal#37]
Table with the following columns:

- name
- dob
- ZIP
- Phone
- added by
- added date

**Comment by leoneln** [comment]
Ill build a prototype for this.. then we can review it.

**Comment by joeldenning** [comment]
👍 sounds good

Suggestion: split this into 4-5 pull requests, instead of one big one. It is enormously easier for me to review several smaller PRs, plus we can find problems and collaborate on it earlier.

The way to split it into smaller pull requests is to push up a small portion of the work and then create a new branch where you keep on working on the code that will be in the next pull request. Then when I review the first one, you go back to the first branch and make changes until we merge it in. Once the first one is merged in, you merge master back into the second branch and push that up as the second code review.

The goal is to go through the entire [github flow](https://guides.github.com/introduction/flow/) more than one time.

Example:
PR 1: Create a new backend API that is only a stub -- doesn't actually call the db yet
PR 2: Implement the backend API
PR 3: Create a new route in the frontend code for the View Clients list
PR 4: Implement the basics of the table
PR 5: Implement advanced parts of the table

**Comment by joeldenning** [comment]
Doing this will allow me to jump in and implement part of it, too. So that we can collaborate on the feature.

It takes practice to learn how to do this ^^. For example, for PR 1 you would need to learn how to call an API with Postman or through the browser console so that you don't have to build the entire UI in order to test the feature.

**Comment by joeldenning** [comment]
Search by ZIP code
Maybe exporting

**Comment by joeldenning** [comment]
See related #99

**Comment by joeldenning** [comment]
Fully implemented now.
